docs/development.adoc

```diff
@@ -160,33 +160,22 @@ $ MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags "--ena
 
 .Procedure:
 
-To deploy an integration test:
+Integration tests can be run in the following two ways:
```

Comment:
Yeah, the below information is completely wrong. You remove information on how to run the tests... (`make test-e2e`) and `make clean-test` doesn't even do anything.

You need to keep the `make test-e2e` here so that people can run these integration tests locally.

The documentation you're adding should be about running each test individually.

Reply:
I have addressed those comment here https://github.com/openshift/odo/pull/1759#issuecomment-496099906. Also i ahev added extra info in the ```NOTE```. Probably the ```NOTE``` section will clarify most of your doubts.